pyop2/codegen/c/inverse.c

```diff
@@ -0,0 +1,24 @@
+#define Inverse_HPP
```

Comment:
This define is unused I think. It was maybe used as an include guard at some point?